logstash-exporter.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: logstash-exporter
+  version: 1.6.1
+  epoch: 0
+  description: Prometheus exporter for Logstash written in Go
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+  environment:
+    GITHUB_REPO: "github.com/kuskoman/logstash-exporter"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 387fb36f814e74e81dd14d82a948e61cefa76f5c
+      repository: https://github.com/kuskoman/logstash-exporter
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      packages: ./cmd/exporter/main.go
+      ldflags: -s -w -X ${GITHUB_REPO}/config.Version="${{package.version}}" -X ${GITHUB_REPO}/config.GitCommit="$(git rev-parse --short HEAD)" -X ${GITHUB_REPO}/config.BuildDate="$(date -u +%Y-%m-%dT%H:%M:%SZ)"
+      repository: https://github.com/kuskoman/logstash-exporter
+      output: main
+
+  - uses: strip
+
+update:
```

Comment:
`wolfictl check update` looks happy, so, it shouldn't be any problem I guess

Reply:
There are pre-releases here:
v2.0.0-pre2 (tag name). Release v2.0.0-pre2 (release name)

Depending which one your checking (i.e tags or releases, then these will need to be filtered out, otherwise it'll build for pre-releases.

There are some examples in wolfi using `ignore-regex-patterns:

Reply:
It'll still include pre-releases though so we'll need to filter those out using ignore-regex. I'll make a suggestion inline below but check if the syntax is correct :)